### DISCUSSION TITLE
Failing Builds iOS React Native

### DISCUSSION THREAD

**Discussion by saintmalik** [codemagic-ci-cd/codemagic-docs#2017] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

we use the default codemagic iOS build workflow, but the build fails in the "Build ipa for distribution"

### Expected results

A build that doesn't fail

### Actual results

Using Xcode 14.3.1 (14E300c)
Check myapp.xcworkspace build settings
Execute "xcodebuild -workspace /Users/builder/clone/ios/myapp.xcworkspace -scheme myapp -showBuildSettings"

Archive myapp.xcworkspace
Execute "xcodebuild -workspace /Users/builder/clone/ios/myapp.xcworkspace -scheme myapp -archivePath build/ios/xcarchive/myapp_8okgt27d.xcarchive archive COMPILER_INDEX_STORE_ENABLE=NO DEVELOPMENT_TEAM=93C44M65VB 'CODE_SIGN_IDENTITY=Apple Distribution' CODE_SIGN_STYLE=Manual"

    Run script build phase 'Start Packager' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'myapp' from project 'myapp')
    Run script build phase 'Bundle React Native code and images' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'myapp' from project 'myapp')
    Run script build phase '[CP-User] [RNFB] Core Configuration' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'myapp' from project 'myapp')
Failed to archive /Users/builder/clone/ios/myapp.xcworkspa

### Build id (optional)

64f5bcfed3d03298ff98c458

**Comment by himesh-cm** [comment]
Hi @saintmalik, Can you share the build ID? The error messages are not providing useful information right now.

**Reply by saintmalik** [reply]
okay

**Reply by saintmalik** [reply]
Was the build link helpful

**Reply by himesh-cm** [reply, marked as answer]
Hi @saintmalik, The issue is with the provisioning profile
```
error: Provisioning profile "okaytestflightprofile" doesn't support the Associated Domains capability.
```

please add it from the apple developer account and try re-running the build.

**Reply by saintmalik** [reply]
sorry which domains capability is it requesting?

**Comment by saintmalik** [comment]
Thank you for your reply

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
